src/ch/epfl/leb/sass/simulator/loggers/FrameLogger.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (C) 2017 Laboratory of Experimental Biophysics
+ * Ecole Polytechnique Federale de Lausanne
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package ch.epfl.leb.sass.simulator.loggers;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+
+/**
+ * Reports the positions of all fluorophores visible in each frame in a file
```

Comment:
You might also want to add a comment about what the time_on column means. I think it means the fraction of the frame that the fluorophore spent in the on-state, i.e. 0 < time_on <= 1

Reply:
I've written a small description of what time_on is when it's used as an argument for the logFrame function, on line 65. I've chosen to present it that way to have the same code layout as the other loggers. Do you think that it makes them difficult to read/understand?